parts/k8s/addons/kubernetesmasteraddons-calico-daemonset.yaml

```diff
@@ -252,6 +252,19 @@ spec:
 
 ---
 
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: calico-typha
+  namespace: kube-system
+spec:
+  minAvailable: 0%
```

Comment:
This sort of seems like a no-op configuration, but I infer that cluster-autoscaler needs *some* value here in order to engage?

Reply:
Absolutely. The cluster-autoscaler will refuse to engage in a scale down if pods are non-mirrored and does not have a PDB that allows the pod to be killed.